docs/Extensions.md

```diff
@@ -27,6 +27,14 @@ services:
 
 For explanations of these extensions, please refer to the [Podman Documentation](https://docs.podman.io/).
 
+## Secrets
+```yml
```

Comment:
Add:

`The following extension keys are available under secret configuration:

* `x-podman.relabel` - Configure SELinux relabeling

For example, the following configures `custom-secret` to use mount with private and unshared content.